nncf/parameters.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
I think it is better to move this file to a separate subpackage `nncf.parameters`

Reply:
Could you briefly explain why?

Reply:
Probably it does not make sense to do this at this stage.